test/unit/specs/Select.spec.js

```diff
@@ -1295,12 +1295,11 @@ describe('Select.vue', () => {
 			})
 		})
 
-		it('should select an existing option if the search string matches an objects label from options when filter-options is false', (done) => {
-			let two = {label: 'two'}
+		it('should select first option if the search string matches an objects label from options when filterable is false', (done) => {
```

Comment:
Not sure what to do with this one. Previously the filter would be applied leaving only the matched option at the top of the list. Now the filter is not longer being applied the option list remains unchanged so the first option would be selected.

The only other way I can think to handle this would be the re-order the options such that the matched option is moved to the top of the filteredOptions list?

Either way it is bit of an edge case -- with `filterable=false` you would expect the server to provide already filtered options.